src/tmt_web/generators/templates/testorplan.html.j2

```diff
@@ -0,0 +1,24 @@
+<!-- test_page.html.j2 -->
+{% extends '_base.html.j2' %}
+{% set title = testorplan.name %}
+
+{% block content %}
+    <p>name: {{ testorplan.name }}</p>
```

Comment:
We should probably improve the design of the page in the future. Right know is very very simple:

![Screenshot 2024-09-19 at 17-44-14 _tests_core_smoke](https://github.com/user-attachments/assets/36df7df1-5d7b-429b-845f-ea7dfe925a1f)

But for the first version, I think it's fine. We can improve it in the future.

Reply:
This will be done by implementing a frontend in future PR, no?